src/client/components/FormGenerator.tsx

```diff
@@ -42,13 +43,12 @@ const FormGenerator: React.FC<FormGeneratorProps> = ({
 
   const handleSubmit: FormEventHandler = async (e) => {
```

Comment:
I know it's not part of your changes. But since the PR is still open, can we remove the async in this handler since it's not doing anything asynchronous?? 😅

Reply:
Totally! Right away